tools/cabana/mainwin.cc

```diff
@@ -25,6 +25,23 @@
 #include "tools/cabana/utils/export.h"
 #include "tools/replay/replay.h"
 
+#include <unistd.h>
+#include <linux/limits.h>
+#include <filesystem>
+
+const char* get_opendbc_file_path(int distance_from_root) {
```

Comment:
this shouldn't be duplicated, make a .h file for sharing this. can we also use something like __file__ rather than hardcoding the distance from root?

Reply:
I really wish I could do that, but __FILE__ is resolved to be relative to the file that calls it (`./tools/cabana` if called from the root directory) so that string would not be viable to be used.